### PR TITLE
Rename test report

### PR DESCRIPTION
**Pull request by beatngu13**
Please be aware that this is only a draft: I just wonder what to do with `XmlMigratorInstances`, including various migrators and tests. Do we still need them? I mean, we do a major release and (intentionally) create breaking changes?

### CODE REVIEW

**Review by martin-v — APPROVED** [review]

**Review by martin-v — APPROVED** [review]

**Review by martin-v — APPROVED** [review]

File: src/main/java/de/retest/recheck/printer/TestReportPrinter.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package de.retest.recheck.printer;
+
+import java.util.stream.Collectors;
+
+import de.retest.recheck.ignore.ShouldIgnore;
+import de.retest.recheck.report.TestReport;
+
+public class TestReportPrinter implements Printer<TestReport> {
```
Comment (martin-v) [inline src/main/java/de/retest/recheck/printer/TestReportPrinter.java:8]: This class is used in?

**Review by martin-v — APPROVED** [review]

File: src/main/java/de/retest/recheck/printer/TestReportPrinter.java (hunk 1 above)
Comment (beatngu13) [reply src/main/java/de/retest/recheck/printer/TestReportPrinter.java:8]: Mostly recheck.cli to create a detailed version of `toString()` (formerly via `toDetailedString()`). We have decoupled these concerns.

**Comment by rebazer** [comment]
Rebase failed, this pull request needs some manual love ...

**Comment by rebazer** [comment]
Rebase failed, this pull request needs some manual love ...

**Comment by rebazer** [comment]
Rebase failed, this pull request needs some manual love ...

**Comment by rebazer** [comment]
Rebase failed, this pull request needs some manual love ...

**Pull request merged**